Firestore/core/src/firebase/firestore/util/string_apple.h

```diff
@@ -61,14 +76,21 @@ inline absl::string_view MakeStringView(NSString* str) {
 
 // Creates a std::string wrapper for the contents of the given NSString.
 inline std::string MakeString(NSString* str) {
+  CFStringRef cf_str = (__bridge CFStringRef)str;
+  return MakeString(cf_str);
+}
+
+// Creates a std::string wrapper for the contents of the given NSString.
+inline std::string MakeStringOld(NSString* str) {
```

Comment:
I presume this was used to get benchmarks. Is it possible to delete this function before merging>

Reply:
I actually get benchmarks by checking out the before and after commits. This is just stale from when I was experimenting. Deleted.